src/cryptoadvance/specter/templates/includes/helpers.jinja

```diff
@@ -29,41 +192,85 @@
+
+
+
+    // Formats the valueInBTC (e.g. from a tx output) to an array
+    // e.g. ["0.22569496", "tBTC"]
+    // by default targetUnit will be set to BTC for liquid to prevent any conversion of valueInBTC
+    function formatBtcAmountAndUnitArray(valueInBTC, targetUnit='{{ "BTC" if specter.is_liquid else specter.unit }}', hideStripped=true){
+        if ('{{ specter.hide_sensitive_info }}' == 'True'){
+            return ["#########"]};
+        if ((valueInBTC < 0 || valueInBTC == null) && '{{ specter.is_liquid }}' == 'True'){
+            return ["Confidential"]}
+        if (valueInBTC == null){
+            return ["Unknown"]};
+
+        const convertToSat = targetUnit == 'sat';
+        var formattedUnitLabel = formatUnitLabel(targetUnit);        
+        var formattedValue = convertToSat ? internalFormatBtcAmountAsSats(valueInBTC) : internalFormatBtcAmount(
+                                                                                            valueInBTC,
+                                                                                            7,
+                                                                                            6,
+                                                                                            true,
+                                                                                            hideStripped);
+        return [formattedValue, formattedUnitLabel];  
     }
 
-    // Determines if an unitLabel (e.g. from a tx output) is "BTC", "LBTC", "tBTC", "tLBTC"
-    function unitLabelIsBitcoin(unitLabel){
-        if (!unitLabel){
-            return true            
-        } 
-        return ([null, "", "LBTC", "BTC", "tBTC", "tLBTC"].indexOf(unitLabel) > -1)
+    // Formats the valueInBTC (e.g. from a tx output) to an "formattedValue formattedUnitLabel"
+    // e.g. "0.22569496 tBTC"
+    function formatBtcAmountAndUnit(valueInBTC, targetUnit='{{ "BTC" if specter.is_liquid else specter.unit }}', hideStripped=true){
+        return formatBtcAmountAndUnitArray(valueInBTC, targetUnit, hideStripped).join(' ');  
     }
+    // Formats the valueInBTC (e.g. from a tx output) to an "formattedValue"
+    // e.g. "0.22569496"
+    function formatBtcAmount(valueInBTC, targetUnit='{{ "BTC" if specter.is_liquid else specter.unit }}', hideStripped=true){
+        return formatBtcAmountAndUnitArray(valueInBTC, targetUnit, hideStripped)[0];  
+    }
+
+
+
 
     // Calculates and formats the price as a span class="note"
-    function formatPrice(valueInBTC, unitLabel, symbol, price){
+    function formatPrice(valueInBTC, 
+                        unit='BTC', 
+                        symbol='{{ specter.alt_symbol }}', 
+                        price='{{ specter.alt_rate }}',
+                        wrapInSpan=true,
+                        ){        
+        {% if not specter.price_check %}
+            return ''
+        {% endif %}
+
+
+        if ('{{ specter.hide_sensitive_info }}' == 'True'){
```

Comment:
This line is causing in Firefox (not in Chromium, though ...):

<img width="448" alt="grafik" src="https://user-images.githubusercontent.com/70536101/192487519-7140310b-ce78-4c6f-8d0e-bb1519674ae1.png">

See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Stmt_after_return

Probably caused by the structure of the semi-colons. Ca

We should really start to use a JS linter ...

Reply:
Should be solved in [80987a4](https://github.com/cryptoadvance/specter-desktop/pull/1901/commits/80987a4c5f75342186b50b78d6719f02c0fe1087)

Reply:
Looks cleaner, I'd say!